react/Example.tsx

```diff
@@ -1,15 +1,18 @@
 import React, { useState } from 'react'
 import PlaceDetails, { LineFragment } from './PlaceDetails'
+import AddressForm from './AddressForm'
 import { Dropdown } from 'vtex.styleguide'
 import {
   countryDescriptions,
   CountryDescription,
   sampleAddress as address,
 } from './mocks/CountryDescriptions'
+import { AddressContextProvider, useAddressContext } from 'vtex.address-context/AddressContext'
 
-const PlaceComponents: StorefrontFunctionComponent<{}> = () => {
+const Example: StorefrontFunctionComponent<{}> = () => {
   const [option, setOption] = useState<string>('')
   const [summary, setSummary] = useState<LineFragment[][]>([])
+  const { address } = useAddressContext()
```

Comment:
Currently, it is the `Example` component that makes the communication with the orchestrator. In reality, it should be the each component (`PlaceDetails`, `AddressForm`, and `LocationForm`) that make this communication, however, since this PR is mainly to make the integration with the orchestrator work, I'll leave that for a later PR.